dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -179,7 +179,7 @@ private OrganisationUnit validateRequestedOrgUnit(String orgUnitUid, User user)
     }
 
     if (!organisationUnitService.isInUserHierarchy(
```

Comment:
Can `user.isInUserEffectiveSearchOrgUnit` be used here and similar usages later on?

Reply:
We could, but first we'd need to replace `User` by `UserDetails`.
That kind of user is present in multiple locations in our validators and I'm not sure that's a refactor we want to do in this ticket.